config/crd/bases/memgraph.com_memgraphhas.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: memgraphhas.memgraph.com
+spec:
+  group: memgraph.com
+  names:
+    kind: MemgraphHA
+    listKind: MemgraphHAList
+    plural: memgraphhas
```

Comment:
This feels very strange, and I am not sure what to do about this. Also, the yaml file name is very strange. I guess this was auto-generated?